src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_BOUND = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public String getRandomBall() {
```

Comment:
we should return Ball instead of String here
```suggestion
    public Ball getRandomBall() {
```